lib/services/http-service.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016, EMC, Inc.
+// Copyright 2015, EMC, Inc.
```

Comment:
From what I captured in Slack discussion, only the newly added files need to update to copyright 2016. @benbp @yyscamper @anhou should know better.